src/js/views/globals/search/patient-search.scss

```diff
@@ -6,7 +6,7 @@
 
 .patient-search__close {
   position: absolute;
-  right: 24px;
+  right: 16px;
```

Comment:
Updated this to match the styling in the design doc. Moves the `X` close icon closer to the right side of the modal.

<img width="549" alt="Screen Shot 2022-09-08 at 12 02 21 PM" src="https://user-images.githubusercontent.com/35355575/189182449-6066d6bf-78e3-4d73-b1ce-32e2190d1095.png">